apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/http/LoggingInterceptor.kt

```diff
@@ -10,7 +10,7 @@ import kotlin.jvm.JvmOverloads
 /**
  * An interceptor that logs requests and responses.
  *
- * @param level the level of logging. Caution: when uploading files, setting this to [Level.BODY] will cause the files to
+ * @param [level] the level of logging. Caution: when uploading files, setting this to [Level.BODY] will cause the files to
```

Comment:
[TIL](https://kotlinlang.org/docs/kotlin-doc.html#throws-class-exception-class)!

Reply:
So it's technically not changing anything, right?

Reply:
> So it's technically not changing anything, right?

Nothing is changing with the brackets, both syntaxes are equal as is the effect on the docs. It just adds a bit of readability here when overviewing the docs inside the source code.

Reply:
Sounds good, thanks!